layer/USAGE_desktop.md

```diff
@@ -1,106 +1,141 @@
-Log File Create New | GFXRECON_LOG_FILE_CREATE_NEW | BOOL | This option indicates that you want to either create a new file every time the layer is triggered, or append to the old log file.
-Log File Flush After Write | GFXRECON_LOG_FILE_FLUSH_AFTER_WRITE | BOOL | This option allows you to force a flush after every log file write to make sure you don't lose messages in a buffer.
-Log File Keep Open | GFXRECON_LOG_FILE_KEEP_OPEN | BOOL | This option forces the log file to remain open after it's created to allow for faster recording of log messages.
-Log Level | GFXRECON_LOG_LEVEL | STRING | This option allows you to choose what log level you desire to trigger.  Available options include: "debug", "info", "warning", "error", and "fatal".  Any level selected will include all levels listed after it.  For example, choosing "warning" will also log out "error" and "fatal" messages.
-Log Output To Console | GFXRECON_LOG_OUTPUT_TO_CONSOLE | BOOL | This option allows log messages to be written out to stdout (or whatever debug console is available on the target platform).
-Log Output To OS Debug String | GFXRECON_LOG_OUTPUT_TO_OS_DEBUG_STRING | BOOL | This option allows log messages to be written out to the OS-specific logging mechanism.  Currently only works for Windows, but allows debug messages to re-direct from the console to `OutputDebugStringA`.
-Memory Tracking Mode | GFXRECON_MEMORY_TRACKING_MODE | STRING | This option allows the user to determine what memory tracking mode the layer uses when handling memory.  Available options are: "page_guard", "assisted" and "unassisted".  <ul><li>"page_guard" is used to determine which regions of memory to write on an `vkUnmapMemory` or `vkQueueSubmit` call.</li><li>"assisted" assumes the application will always flush after writing to mapped memory, so will only write on a flush.</li><li>"unassisted" assumes the application does not flush, so writes all mapped data on an `vkUnmapMemory` or `vkQueueSubmit` call.</li></ul>
-
-## Defining the Output Location
-
-By default, the layer will generate a file called `gfxrecon_capture.gfxr` in
-the current working directory.
-However, you may define the environment variable `GFXRECON_CAPTURE_FILE` on
-desktop to indicate the location and name of the resulting capture file
-as defined above in [GFXRecon Debug Options](#gfxrecon-debug-options).
-
-## Using
-
-Simply run the Vulkan application as normal after defining the above items.
+Capture File Name | GFXRECON_CAPTURE_FILE | STRING | Path to use when creating the capture file.  Default is: `gfxrecon_capture.gfxr`
+Capture File Compression Type | GFXRECON_CAPTURE_COMPRESSION_TYPE | STRING | Compression format to use with the capture file.  Valid values are: `LZ4`, `ZLIB`, and `NONE`. Default is: `LZ4`
+Capture File Timestamp | GFXRECON_CAPTURE_FILE_TIMESTAMP | BOOL | Add a timestamp to the capture file with the form: "gfxrecon_capture_yyyymmddThhmmss.gfxr".  Default is: `true`
```

Comment:
Oh, and:

"gfxrecon_capture_yyyymmddThhmmss.gfxr" => `gfxrecon_capture_yyyymmddThhmmss.gfxr`